### DISCUSSION TITLE
Why in the prerequisites part of Free-Hit Node.Js is mentioned?

### DISCUSSION THREAD

**Discussion by suzy-g38** [JasonDsouza212/free-hit#131] [General]
As far as I can understand this project is complete;y about ReactJs, with pnpm, vite. Git as a prerequisite, I can understand but I am not understanding why Node.Js is mentioned?
If anyone can explain it that will be very much helpful.
Thank you.

**Comment by JasonDsouza212** [comment]
During development, Node.js also allows for managing dependencies, and it includes the npm package manager

**Reply by suzy-g38** [reply]
> During development, Node.js also allows for managing dependencies, and it includes the npm package manager

okay okay, got it.
